functions_framework_tool/templates/helloworld/README.md

```diff
@@ -0,0 +1,111 @@
+# Hello world example
+
+This example handles HTTP GET requests by responding with 'Hello, World!'.
```

Comment:
Seems like it handles cloudevents.

Reply:
Sorry, not following you here...? This example is meant to be analogous to examples we have for other languages, like this [one](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/functions/helloworld/index.js#L47), for example.